datapackage_pipelines_aws/processors/change_acl.py

```diff
@@ -0,0 +1,23 @@
+import os
+import boto3
+
+from datapackage_pipelines.wrapper import ingest, spew
+
+parameters, datapackage, resource_iterator = ingest()
+
+
+def change_acl():
+    endpoint_url = os.environ.get("S3_ENDPOINT_URL")
+    s3 = boto3.client('s3', endpoint_url=endpoint_url)
+    bucket = parameters['bucket']
+    key = parameters.get('path', '')
+    acl = parameters['acl']
+    objs = s3.list_objects(Bucket=bucket, Prefix=key)
+    keys = [content['Key'] for content in objs['Contents']]
+    for obj in keys:
```

Comment:
do we get the ACL when listing objects?

Reply:
@akariv no I don't think so. do we need it? 
This just takes everything with given prefix and Changes ACL